codegen-core/src/main/kotlin/software/amazon/smithy/rust/codegen/core/testutil/EventStreamTestTools.kt

```diff
@@ -118,11 +119,15 @@ object EventStreamTestTools {
         val symbolProvider = codegenContext.symbolProvider
         val operationShape = model.expectShape(ShapeId.from("test#TestStreamOp")) as OperationShape
         val unionShape = model.expectShape(ShapeId.from("test#TestStream")) as UnionShape
+        val walker = DirectedWalker(model)
 
         val project = TestWorkspace.testProject(symbolProvider)
         val operationSymbol = symbolProvider.toSymbol(operationShape)
         project.withModule(ErrorsModule) {
-            val errors = model.structureShapes.filter { shape -> shape.hasTrait<ErrorTrait>() }
+            val errors = model.serviceShapes
```

Comment:
@jdisanti What do you think about replacing these tests:

```
ClientEventStreamUnmarshallerGeneratorTest.kt
ServerEventStreamUnmarshallerGeneratorTest.kt
ClientEventStreamMarshallerGeneratorTest.kt
ServerEventStreamMarshallerGeneratorTest.kt
```

by simple `{client,server}IntegrationTests`? (see https://github.com/awslabs/smithy-rs/pull/1956/) They're notoriously hard to get right and a sizable amount of code we could delete.